docs/source/using/experimental-features.adoc

```diff
@@ -303,3 +303,16 @@ Create the file `/etc/resolver/nip.io` with the following content:
 nameserver <ip_address_of_the_minishfit_vm>
 search_order 1
 ----
+
+[[systemtray]]
+== Minishift system tray
+
+To help the users of {project} on macOS and Windows perform simple tasks like starting and stoppping a profile from the GUI, an experimental system tray
+is included in the macOS and Windows binaries. By default the tray is auto started upon minishift start.
```

Comment:
> an experimental system tray is included in the macOS and Windows binaries. 

I do not think this is correct technically because at this point user need to compile it with `make systemtray`. Please update the document to reflect that.

Reply:
are we not going to add the binaries-with-systemtray in this release, as we planned before?